src/BaseConnection.php

```diff
@@ -180,6 +180,34 @@ function( PDOStatement $statement ) {
         );
     }
 
+    public function getAllMulti( $statement, array $params = [] ): array {
```

Comment:
Can you add a DocBlock? And list these new functions in the README?

Reply:
I could, but they're harder and less interesting to do when you're drunk :)